pkg/deps/result.go

```diff
@@ -31,10 +31,12 @@ const (
 
 // Result is a single item that represents a resolved dependency license.
 type Result struct {
-	Dependency      string
-	LicenseFilePath string
-	LicenseContent  string
-	LicenseSpdxID   string
+	Dependency       string
+	LicenseFilePath  string
+	LicenseContent   string
+	LicenseSpdxID    string
+	ManifestFilePath string
```

Comment:
This seems to be unused now, do you think it will be used any soon?

Reply:
Nope. The `ManifestFilePath` field seems redundent. I'm gonna delete this field.